packages/@react-spectrum/s2/src/Table.tsx

```diff
@@ -11,85 +11,1110 @@
+          id: 'sort-desc'
+        },
+        ...options
+      ];
+    }
+    return options;
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [allowsSorting]);
+
+
+  let buttonAlignment = 'start';
+  let menuAlign = 'start' as 'start' | 'end';
+  // TODO: align center is quite strange, copied this from S1 but really there isn't a good place to put the menu when the column text is centered
+  if (align === 'center') {
+    buttonAlignment = 'center';
+  } else if (align === 'end') {
+    buttonAlignment = 'end';
+    menuAlign = 'end';
+  }
 
   return (
-    <AriaRow id={id} {...otherProps}>
-      {allowsDragging && (
-        <Cell>
-          <Button slot="drag">≡</Button>
-        </Cell>
+    <>
+      <MenuTrigger align={menuAlign}>
+        <Button className={(renderProps) => resizableMenuButtonWrapper({...renderProps, align: buttonAlignment})}>
+          {allowsSorting && (
+            <Provider
+              values={[
+                [IconContext, {
+                  styles: sortIcon({isButton: true})
+                }]
+              ]}>
+              {sortDirection != null && (
+                sortDirection === 'ascending' ? <SortUpArrow /> : <SortDownArrow />
+              )}
+            </Provider>
+          )}
+          <div className={columnHeaderText}>
+            {children}
+          </div>
+          <Chevron size="M" className={chevronIcon} />
+        </Button>
+        <Menu onAction={onMenuSelect} items={items}>
+          {(item) => <MenuItem>{item?.label}</MenuItem>}
+        </Menu>
+      </MenuTrigger>
+      <div data-react-aria-prevent-focus="true">
+        <ColumnResizer data-react-aria-prevent-focus="true" className={({resizableDirection, isResizing}) => resizerHandleContainer({resizableDirection, isResizing, isHovered: isInResizeMode || isHovered})}>
+          {({isFocusVisible, isResizing}) => (
+            <>
+              <ResizerIndicator isInResizeMode={isInResizeMode} isFocusVisible={isFocusVisible} isHovered={isHovered} isResizing={isResizing} />
+              {(isFocusVisible || isInResizeMode) && isResizing && <div className={nubbin}><Nubbin /></div>}
+            </>
+        )}
+        </ColumnResizer>
+      </div>
+    </>
+  );
+}
+
+function ResizerIndicator({isFocusVisible, isHovered, isResizing, isInResizeMode}) {
+  let state = useContext(ResizableTableContainerContext);
+
+  return (
+    <div style={{height: isResizing ? state?.tableHeight : '100%'}} className={resizerHandle({isFocusVisible, isHovered: isHovered || isInResizeMode, isResizing})} />
+  );
+}
+
+const tableHeader = style({
+  height: 'full',
+  width: 'full',
+  backgroundColor: 'gray-75'
+});
+
+const selectAllCheckbox = style({
+  marginStart: 16, // table-edge-to-content, same between mobile and desktop
+  marginEnd: {
+    default: 8,
+    scale: {
+      large: 14
+    }
+  }
+});
+
+const selectAllCheckboxColumn = style({
+  padding: 0,
+  height: '[calc(100% - 1px)]',
```

Comment:
```suggestion
  height: '[calc(100% - self(borderBottomWidth))]',
```
is this calculation actually this?

Note: this may have a bug if you try to do it, see my PR internally for more information. I'll open it here in the near future

Reply:
which PR specifically? Mind DMing it to me?